jax/_src/api.py

```diff
@@ -333,15 +330,78 @@ def jit(
     DeviceArray([   0,    1,  256, 6561], dtype=int32)
   """
   if FLAGS.experimental_cpp_jit and not config.jax_dynamic_shapes:
-    return _cpp_jit(fun, static_argnums, static_argnames, device, backend,
+    return _jit("cpp", fun, static_argnums, static_argnames, device, backend,
                     donate_argnums, inline, keep_unused)
   else:
-    return _python_jit(fun, static_argnums, static_argnames, device, backend,
+    return _jit("python", fun, static_argnums, static_argnames, device, backend,
                        donate_argnums, inline, keep_unused, abstracted_axes)
 
+def _jit(
+    jit_implementation: Literal["cpp", "python", None],
+    fun: Callable,
+    static_argnums: Union[int, Iterable[int], None] = None,
+    static_argnames: Union[str, Iterable[str], None] = None,
+    device: Optional[xc.Device] = None,
+    backend: Optional[str] = None,
+    donate_argnums: Union[int, Iterable[int]] = (),
+    inline: bool = False,
+    keep_unused: bool = False,
+    abstracted_axes: Optional[Any] = None,
+  ) -> stages.Wrapped:
+  # Implemements common logic between CPP and Python backends
+  _check_callable(fun)
+
+  # Coerce input
+  donate_argnums = _ensure_index_tuple(donate_argnums)
+
+  try:
+    sig = inspect.signature(fun)
+
```

Comment:
Nit: remove newline.

(In general we tend to like a pretty compact style.)

Reply:
Done